starter-code/css/main.css

```diff
@@ -0,0 +1,110 @@
+body{
+  margin:5% 20% 5% 4%;
+  font-family: 'Lato', sans-serif;
+}
+
+header h1{
+  font-family: 'Lato', sans-serif;
+  font-weight: 300;
+  font-size:40px;
+  margin: 0 auto;
+}
+
+header h1 a{
+  text-decoration: none;
+  color:#FAD7A0;
+  font-size:50px;
+}
+
+header h2{
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  margin: 5px 0;
+  font-size: 25px;
+}
+
+header nav{
+  padding:5px 0;
+}
+
+header nav a{
+  color:#DDDDDD;
+  padding:25px 0 0 0;
+  letter-spacing: 1px;
+  border-bottom: 1px solid #DDDDDD;
+  text-decoration: none;
+  margin: 0 10px 5px 0;
+}
+
+header nav a:hover {
+  color: #ABB544;
+  border-bottom: 1px solid #ABB544;
+ }
+
+header nav a:first-child{
+  color:#DDDDDD;
+  padding:20px 0 0 0;
```

Comment:
Another way to write this since you're only adding padding to the left is `padding-left: 20px;`